test/functional/activation_key/add_host_collection_test.rb

```diff
@@ -20,10 +20,12 @@ module HostCollection
         end
 
         it 'allows resolving host collection name' do
-          ex = api_expects(:host_collections, :index) do |p|
-            p['name'] == 'hc3' && p['organization_id'] == 1
+          2.times do
```

Comment:
I'm concerned that this is a bug. I don't know why this API called be happening twice now

Reply:
Yeah, seems like a regression to me. There are supported methods to test for this if we have to: `api_expects(...).at_most(2)`. I think they come from Mocha, but I'm not 100% sure https://mocha.jamesmead.org/Mocha/Expectation.html